src/controllers/root.ts

```diff
@@ -1,26 +1,9 @@
 import { Request, Response } from 'express';
 import endpointsJson from '../endpoints.json';
 
-export function index(request: Request, response: Response) {
-	response.status(200).json({
-		success: true,
-	});
-}
-
 export function endpoints(request: Request, response: Response) {
-	const data = {};
-	endpointsJson.forEach((endpoint) => {
-		const copy = JSON.parse(JSON.stringify(endpoint));
-		delete copy.path;
-		Object.assign(data, {
-			[endpoint.path]: {
-				...copy,
-			},
-		});
-	});
```

Comment:
Could we re-add this so the endpoints are returned on an Object where the key is the route? I think it'd be easier to find the route we need as right now it's an issue of searching through an array of objects.

Reply:
I've implemented this but by changing the JSON file itself (https://github.com/clowder-cloud/backend/pull/36/commits/d1c5901d8f43ba9d25f2055fe9e3e6d57c47a630) following the principle that less-code is better. Any problems with doing it that way?